letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -367,6 +367,37 @@ BootstrapRpmCommon() {
   fi
 }
 
+BootstrapAlpineCommon() {
+  # Tested with:
+  #   - Alpine 3.4
+
+  $SUDO apk update || echo apk update hit problems but continuing anyway...
```

Comment:
You can `cat /etc/alpine-release` or `apk --version` to see the release and add the flag if needed.